### ISSUE TITLE
VS Code showing potential issues in --verbose mode

### ISSUE THREAD

**Issue by adrianocr** [killalau/vscode-liquid-snippets#2]
This is the message

```
[16026:0711/215009.563394:INFO:CONSOLE(28)] "[/Users/adriano/.vscode/extensions/killalau.vscode-liquid-snippets-0.2.6]: The "Tag if"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/electron-browser/workbench.main.js (28)
[16026:0711/215009.563776:INFO:CONSOLE(28)] "[/Users/adriano/.vscode/extensions/killalau.vscode-liquid-snippets-0.2.6]: The "Tag elsif"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/electron-browser/workbench.main.js (28)
[16026:0711/215009.563805:INFO:CONSOLE(28)] "[/Users/adriano/.vscode/extensions/killalau.vscode-liquid-snippets-0.2.6]: The "Tag if else"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/electron-browser/workbench.main.js (28)
[16026:0711/215009.564088:INFO:CONSOLE(28)] "[/Users/adriano/.vscode/extensions/killalau.vscode-liquid-snippets-0.2.6]: The "Tag unless"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/electron-browser/workbench.main.js (28)
[16026:0711/215009.564356:INFO:CONSOLE(28)] "[/Users/adriano/.vscode/extensions/killalau.vscode-liquid-snippets-0.2.6]: The "Tag case"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/electron-browser/workbench.main.js (28)
[16026:0711/215009.564609:INFO:CONSOLE(28)] "[/Users/adriano/.vscode/extensions/killalau.vscode-liquid-snippets-0.2.6]: The "Tag when"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/electron-browser/workbench.main.js (28)
[16026:0711/215009.564923:INFO:CONSOLE(28)] "[/Users/adriano/.vscode/extensions/killalau.vscode-liquid-snippets-0.2.6]: The "Tag for"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/electron-browser/workbench.main.js (28)
[16026:0711/215009.565376:INFO:CONSOLE(28)] "[/Users/adriano/.vscode/extensions/killalau.vscode-liquid-snippets-0.2.6]: The "Tag tablerow"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/electron-browser/workbench.main.js (28)
[16026:0711/215009.565621:INFO:CONSOLE(28)] "[/Users/adriano/.vscode/extensions/killalau.vscode-liquid-snippets-0.2.6]: The "Tag assign"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/electron-browser/workbench.main.js (28)
[16026:0711/215009.565765:INFO:CONSOLE(28)] "[/Users/adriano/.vscode/extensions/killalau.vscode-liquid-snippets-0.2.6]: The "Tag increment"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/electron-browser/workbench.main.js (28)
[16026:0711/215009.565948:INFO:CONSOLE(28)] "[/Users/adriano/.vscode/extensions/killalau.vscode-liquid-snippets-0.2.6]: The "Tag decrement"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/electron-browser/workbench.main.js (28)
[16026:0711/215009.566151:INFO:CONSOLE(28)] "[/Users/adriano/.vscode/extensions/killalau.vscode-liquid-snippets-0.2.6]: The "Tag capture"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/electron-browser/workbench.main.js (28)
[16026:0711/215009.566284:INFO:CONSOLE(28)] "[/Users/adriano/.vscode/extensions/killalau.vscode-liquid-snippets-0.2.6]: The "Tag include"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/electron-browser/workbench.main.js (28)
[16026:0711/215009.566540:INFO:CONSOLE(28)] "[/Users/adriano/.vscode/extensions/killalau.vscode-liquid-snippets-0.2.6]: The "Tag include with parameters"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/electron-browser/workbench.main.js (28)
[16026:0711/215009.566656:INFO:CONSOLE(28)] "[/Users/adriano/.vscode/extensions/killalau.vscode-liquid-snippets-0.2.6]: The "Tag raw"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/electron-browser/workbench.main.js (28)
[16026:0711/215009.566993:INFO:CONSOLE(28)] "[/Users/adriano/.vscode/extensions/killalau.vscode-liquid-snippets-0.2.6]: The "Tag paginate"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/electron-browser/workbench.main.js (28)
[16026:0711/215009.567191:INFO:CONSOLE(28)] "[/Users/adriano/.vscode/extensions/killalau.vscode-liquid-snippets-0.2.6]: The "Filter concat"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/electron-browser/workbench.main.js (28)
[16026:0711/215009.567367:INFO:CONSOLE(28)] "[/Users/adriano/.vscode/extensions/killalau.vscode-liquid-snippets-0.2.6]: The "Filter map"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/electron-browser/workbench.main.js (28)
[16026:0711/215009.567487:INFO:CONSOLE(28)] "[/Users/adriano/.vscode/extensions/killalau.vscode-liquid-snippets-0.2.6]: The "Filter sort"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/electron-browser/workbench.main.js (28)
[16026:0711/215009.567699:INFO:CONSOLE(28)] "[/Users/adriano/.vscode/extensions/killalau.vscode-liquid-snippets-0.2.6]: The "Filter img tag with parameters"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/electron-browser/workbench.main.js (28)
[16026:0711/215009.568019:INFO:CONSOLE(28)] "[/Users/adriano/.vscode/extensions/killalau.vscode-liquid-snippets-0.2.6]: The "Filter append"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/electron-browser/workbench.main.js (28)
[16026:0711/215009.568209:INFO:CONSOLE(28)] "[/Users/adriano/.vscode/extensions/killalau.vscode-liquid-snippets-0.2.6]: The "Filter pluralize"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/electron-browser/workbench.main.js (28)
[16026:0711/215009.568364:INFO:CONSOLE(28)] "[/Users/adriano/.vscode/extensions/killalau.vscode-liquid-snippets-0.2.6]: The "Filter prepend"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/electron-browser/workbench.main.js (28)
[16026:0711/215009.568533:INFO:CONSOLE(28)] "[/Users/adriano/.vscode/extensions/killalau.vscode-liquid-snippets-0.2.6]: The "Filter remove"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/electron-browser/workbench.main.js (28)
[16026:0711/215009.568704:INFO:CONSOLE(28)] "[/Users/adriano/.vscode/extensions/killalau.vscode-liquid-snippets-0.2.6]: The "Filter remove first"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/electron-browser/workbench.main.js (28)
[16026:0711/215009.568860:INFO:CONSOLE(28)] "[/Users/adriano/.vscode/extensions/killalau.vscode-liquid-snippets-0.2.6]: The "Filter replace"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/electron-browser/workbench.main.js (28)
[16026:0711/215009.569047:INFO:CONSOLE(28)] "[/Users/adriano/.vscode/extensions/killalau.vscode-liquid-snippets-0.2.6]: The "Filter replace first"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/electron-browser/workbench.main.js (28)
[16026:0711/215009.569244:INFO:CONSOLE(28)] "[/Users/adriano/.vscode/extensions/killalau.vscode-liquid-snippets-0.2.6]: The "Filter slice single character"-snippet very likely confuses snippet-variables and snippet-placeholders. See https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax for more details.", source: file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/electron-browser/workbench.main.js (28)
```

**Comment by killalau** [comment]
Thank you for reporting this.
How can I get this message? I cannot find this message in output console (i've start VS Code in terminal with `code --verbose .`)

**Comment by adrianocr** [comment]
@killalau  Do you have the latest version of Code? There was an update yesterday (June 11th, 2017). And yes I got the error messages via `code --verbose` while I was trying to debug another extension.

**Comment by killalau** [comment]
Thank you for your feedback. It seems that VS Code has change its snippet rules and I didn't write liquid for a long time so i didn't realize it.
It is fixed in vscode-liquid-snippets `v2.7`, please update your extension.
